### ISSUE TITLE
read-only milvus instance try to cleanup segments on startup

### ISSUE THREAD

**Issue by hawkphantomnet** [milvus-io/milvus#5743]
**Please state your issue using the following template and, most importantly, in English.**
read-only milvus instance try to cleanup segments on startup

**Describe the bug**
we deploy read-only milvus instances by k8s deployment(replicas=5). Some instances (2 of 5) try to clean up some segments when milvus is started and initializing.

**Steps/Code to reproduce behavior**
Not eady to reproduce, there are info/debug logs below.

**Expected behavior**

**Method of installation**
Docker/cpu

**Environment details**
- Hardware/Software conditions (OS, CPU, GPU, Memory)
centos 7

- Milvus version (master or released version)
0.10.4

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
version: 0.5
network:
  bind.address: 0.0.0.0
  bind.port: 19000
  http.enable: true
  http.port: 19121
cluster:
  enable: true
  role: ro
general:
  timezone: UTC+8
  meta_uri: mysql://user:password@host:3308/database
storage:
  path: /var/milvus
  auto_flush_interval: 1
cache:
  cache_size: 120GB
  insert_buffer_size: 1GB
engine_config:
  use_blas_threshold: 800
  gpu_search_threshold: 1000
wal:
  enable: true
  path: /apps/dat/data/wal
logs:
  path: /apps/logs/boot/
```

**Screenshots**

**Additional context**

**Comment by shengjun1985** [comment]
It is duplicate with #4862 which is fixed in v1.1.0.
Please update your Milvus server and try again.

**Comment by shengjun1985** [comment]
If you have more questions, please continue to contact our community.
